app/src/main/java/org/jellyfin/androidtv/auth/ApiBinder.kt

```diff
@@ -23,8 +23,9 @@ class ApiBinder(
 ) {
 	fun updateSession(session: Session?, resultCallback: (Boolean) -> Unit) {
 		if (session == null) {
+			Timber.d("Session is null, resetting current user.")
 			application.currentUser = null
-			resultCallback(false)
+			resultCallback(true)
```

Comment:
Doing this could cause the StartupActivity to wait for a not-null currentUser here:

https://github.com/jellyfin/jellyfin-androidtv/blob/master/app/src/main/java/org/jellyfin/androidtv/ui/startup/StartupActivity.kt#L78-L82

Instead, we should probably check in the callback if the currentUser is set instead of using this boolean:

https://github.com/jellyfin/jellyfin-androidtv/blob/master/app/src/main/java/org/jellyfin/androidtv/auth/SessionRepository.kt#L92

Reply:
Would it ever hit this condition since the session is `null` though?

https://github.com/jellyfin/jellyfin-androidtv/blob/master/app/src/main/java/org/jellyfin/androidtv/ui/startup/StartupActivity.kt#L71

Reply:
No actually it shouldn't be able to hit that condition :p

I took another look at the code (I already forgot how it worked...) and this is indeed the fix to the issue, I shouldn't write code or review when I'm half asleep 😂 .

Reply:
The logging statement doesn't seem necessary to me though